graphistry/dep_manager.py

```diff
@@ -0,0 +1,36 @@
+import importlib
+
+class DepManager:
+    def __init__(self):
+        self.pkgs = {}
+
+    def __getattr__(self, pkg:str):
+        if '_' not in pkg:
+            self._add_deps(pkg)
+            try:
+                return True, "ok", self.pkgs[pkg], self.pkgs[pkg].__version__
+            except KeyError:
+                return False, str(pkg) + " not installed", None, None
```

Comment:
Return the import exn for case of installed but uninportable, eg, using a GPU container lib when on a CPU box